Lib/symbol.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/python3.6
+
+"""Non-terminal symbols of Python grammar (from "graminit.h")."""
+
```

Comment:
This module I would leave out, as it is specific to how the grammar works in cpython.